HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -10930,6 +11559,74 @@ class Pool < BaseElement
              :heater_monthly_multipliers] # [String] Heater/MonthlyScheduleMultipliers
     attr_accessor(*ATTRS)
 
+    # Returns any branch circuits that the component may be attached to.
+    #
+    # @return [Array<HPXML::BranchCircuit>] List of branch circuit objects
+    def pump_branch_circuits
+      list = []
+      @parent_object.electric_panels.each do |electric_panel|
+        electric_panel.branch_circuits.each do |branch_circuit|
+          next if branch_circuit.component_idrefs.nil? || branch_circuit.component_idrefs.empty?
```

Comment:
Why are there 8 methods here and below that don't reference `HPXML::branch_circuits` and `HPXML::service_feeders`?